core/templates/dev/head/services/searchService.js

```diff
@@ -67,16 +67,24 @@ oppia.factory('searchService', [
   var updateSearchFields = function(itemsType, urlComponent,
                                     selectionDetails) {
     var itemCodeGroup = urlComponent.match(/=\("[A-Za-z%20" ]+"\)/);
+    var itemCodes = itemCodeGroup ? itemCodeGroup[0] : null;
 
-    if (itemCodeGroup == undefined) {
+    var EXPECTED_PREFIX = '=("';
+    var EXPECTED_SUFFIX = '")';
+
+    if (!itemCodes ||
+        itemCodes.indexOf(EXPECTED_PREFIX) !== 0 ||
```

Comment:
IE has no support for startsWith, unfortunately ... https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith